hsctl

```diff
@@ -369,6 +372,7 @@ rebuild_hs() {
     rebuild_solr_index
     sleep 2s
     restart_hs
+    sleep 2s
```

Comment:
All of these sleeps... are any of them necessary?

Reply:
Those have always been there.  I did not add any extra sleeps to hsctl.

Reply:
There are like 10 seconds of sleep added in this diff.